addons/web_editor/views/snippets.xml

```diff
@@ -460,78 +460,156 @@
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_heptagon" data-select-label="Heptagon (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_star" data-select-label="Star (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_star_7pin" data-select-label="Star 2 (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_star_8pin" data-select-label="Star 3 (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_star_16pin" data-select-label="Star 4 (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_emerald" data-select-label="Emerald (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_lemon" data-select-label="Lemon (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_tear" data-select-label="Tear (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_pill" data-select-label="Pill (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_gem" data-select-label="Gem (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_cornered2" data-select-label="Cornered 2"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_door" data-select-label="Door (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_sonar" data-select-label="Sonar (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_clover" data-select-label="Clover (R)"/>
+                        <we-button data-set-img-shape="web_editor/geometric_round/geo_round_bread" data-select-label="Bread (R)"/>
+                        <t t-if="not no_animation">
+                            <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/geo_round_square_1" data-select-label="Square 10 (R)" data-animated="true"/>
+                            <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/geo_round_square_2" data-select-label="Square 12 (R)" data-animated="true"/>
+                            <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/geo_round_blob_soft" data-select-label="Blob Soft" data-animated="true"/>
+                            <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/geo_round_blob_medium" data-select-label="Blob Medium" data-animated="true"/>
+                            <we-button t-if="not no_animations" data-set-img-shape="web_editor/geometric_round/geo_round_blob_hard" data-select-label="Blob Hard" data-animated="true"/>
```

Comment:
Forgot to remove those `no_animations` (you added it to the parent `<t/>`)

Reply:
Good catch, sorry

For the latest push I noticed that I forgot to adapt the old shapes width and height so I also changed the svg width and height to be in accordance with the new shape (600x600)